engine/src/flutter/lib/web_ui/lib/src/engine/semantics/table.dart

```diff
@@ -34,7 +34,7 @@ class SemanticCell extends SemanticRole {
     : super.withBasics(
         EngineSemanticsRole.cell,
         semanticsObject,
-        preferredLabelRepresentation: LabelRepresentation.ariaLabel,
+        preferredLabelRepresentation: LabelRepresentation.sizedSpan,
```

Comment:
Can you please leave a comment explaining this choice? It would also be useful to link to the github issue in the comment.